src/mixxxapplication.cpp

```diff
@@ -28,7 +28,11 @@ Q_IMPORT_PLUGIN(QWindowsVistaStylePlugin)
 Q_IMPORT_PLUGIN(QCocoaIntegrationPlugin)
 Q_IMPORT_PLUGIN(QMacStylePlugin)
 #else
-#error "Q_IMPORT_PLUGIN() for the current patform is missing"
```

Comment:
For a more robust solution, please keep the original else branch and check for Linux explicit.

Reply:
I did initially but then I realized that would exclude BSD.

Reply:
In case both use the same plug-ins, we should make that explicit.